src/modules/controllers/task.controller.js

```diff
@@ -0,0 +1,42 @@
+const Task = require('../task')
```

Comment:
src/modules/controllers/task.controller.js
Call not through a dot, but through a hyphen:
task-controller.js